data/models.py

```diff
@@ -0,0 +1,194 @@
+    SOURCE = (
+        SUGGESTION_SOURCE,
+        "ingestion de source de données",
+    )
+    SOURCE_AJOUT = (
+        SUGGESTION_SOURCE_AJOUT,
+        "ingestion de source de données - nouveau acteur",
+    )
+    SOURCE_MISESAJOUR = (
+        SUGGESTION_SOURCE_MISESAJOUR,
+        "ingestion de source de données - modification d'acteur existant",
+    )
+    SOURCE_SUPPRESSION = SUGGESTION_SOURCE_SUPRESSION, "ingestion de source de données"
+    SOURCE_ENRICHISSEMENT = SUGGESTION_ENRICHISSEMENT, "suggestion d'enrichissement"
+
+
+class SuggestionCohorte(models.Model):
+    id = models.AutoField(primary_key=True)
+    # On utilise identifiant car le champ n'est pas utilisé pour résoudre une relation
+    # en base de données
+    identifiant_action = models.CharField(
+        max_length=250, help_text="Identifiant de l'action (ex : dag_id pour Airflow)"
+    )
+    identifiant_execution = models.CharField(
+        max_length=250,
+        help_text="Identifiant de l'execution (ex : run_id pour Airflow)",
+    )
+    type_action = models.CharField(
+        choices=SuggestionAction.choices,
+        max_length=250,
+        blank=True,
+    )
+    statut = models.CharField(
+        max_length=50,
+        choices=SuggestionStatut.choices,
+        default=SuggestionStatut.AVALIDER,
+    )
+    metadata = models.JSONField(
+        null=True, blank=True, help_text="Metadata de la cohorte, données statistiques"
+    )
+    pourcentage_erreurs_tolerees = models.IntegerField(
+        default=0,
+        db_default=0,
+        help_text="Nombre d'erreurs tolérées en pourcentage",
+        validators=[MinValueValidator(0), MaxValueValidator(100)],
+    )
+    cree_le = models.DateTimeField(auto_now_add=True, db_default=Now())
+    modifie_le = models.DateTimeField(auto_now=True, db_default=Now())
+
+    @property
+    def is_source_type(self) -> bool:
+        # FIXME: ajout de tests
+        return self.type_action in [
+            SuggestionAction.SOURCE,
+            SuggestionAction.SOURCE_AJOUT,
+            SuggestionAction.SOURCE_MISESAJOUR,
+            SuggestionAction.SOURCE_SUPPRESSION,
+        ]
+
+    @property
+    def is_clustering_type(self) -> bool:
+        # FIXME: ajout de tests
+        return self.type_action == SuggestionAction.CLUSTERING
+
+    def __str__(self) -> str:
+        return f"{self.identifiant_action} - {self.identifiant_execution}"
+
+    def display_meta_data(self) -> dict:
+        displayed_metadata = {}
+        displayed_metadata["Nombre d'acteur à créer ou mettre à jour"] = (
+            self.metadata.get("acteurs_to_add_or_update", 0)
+        )
+        displayed_metadata["Nombre de duplicats"] = self.metadata.get(
+            "number_of_duplicates", 0
+        )
+        displayed_metadata["Nombre d'acteur à supprimer"] = self.metadata.get(
+            "number_of_removed_actors", 0
+        )
+        return displayed_metadata
+
+
+class SuggestionUnitaire(models.Model):
```

Comment:
```suggestion
class Suggestion(models.Model):
```

Cf discussion, permettrait p-e de rendre le fonctionnement plus clair à la première lecture des modèles Django